frontend/src/components/CopyToClipboard.js

```diff
@@ -0,0 +1,37 @@
+import ContentCopyOutlined from "@mui/icons-material/ContentCopyOutlined";
+import { Box, Chip, IconButton, Tooltip } from "@mui/material";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { copyToClipboard } from "../utils/clipboard";
+import { shortenAddress } from "../utils/util";
+
+export const CopyToClipboard = (props) => {
+  const { message } = props;
+  const toolTipEnabled = props.toolTipEnabled;
```

Comment:
```suggestion
  const { message, enableTooltip } = props;
```

Reply:
const { message, enableTooltip } throws error if enableTooltip is not in props

Reply:
done